packages/requisitions/src/CustomerRequisition/DetailView/Footer.tsx

```diff
@@ -0,0 +1,143 @@
+import {
+  Box,
+  ArrowRightIcon,
+  ButtonWithIcon,
+  SaveIcon,
+  StatusCrumbs,
+  XCircleIcon,
+  useTranslation,
+  useNotification,
+  AppFooterPortal,
+} from '@openmsupply-client/common';
+import React, { FC } from 'react';
+import { useNavigate } from 'react-router-dom';
+import {
+  getSupplierRequisitionStatuses,
+  getNextSupplierRequisitionStatus,
+  getSupplierRequisitionTranslator,
+  isRequisitionEditable,
+} from '../../utils';
+import { CustomerRequisition } from '../../types';
+
+interface OutboundDetailFooterProps {
+  draft: CustomerRequisition;
+  save: () => Promise<void>;
+}
+
+const getNextStatusText = (draft: CustomerRequisition) => {
+  const nextStatus = getNextSupplierRequisitionStatus(draft.status);
+  const translation = getSupplierRequisitionTranslator()(nextStatus);
+  return translation;
+};
+
+const createStatusLog = (
+  status: 'DRAFT' | 'IN_PROGRESS' | 'FINALISED' | 'SENT'
+) => {
+  if (status === 'DRAFT') {
+    return {
+      DRAFT: new Date().toISOString(),
+      IN_PROGRESS: null,
+      FINALISED: null,
+      SENT: null,
+    };
+  }
+  if (status === 'IN_PROGRESS') {
+    return {
+      DRAFT: new Date().toISOString(),
+      IN_PROGRESS: new Date().toISOString(),
+      FINALISED: null,
+      SENT: null,
+    };
+  }
+
+  if (status === 'FINALISED') {
+    return {
+      DRAFT: new Date().toISOString(),
+      IN_PROGRESS: new Date().toISOString(),
+      FINALISED: new Date().toISOString(),
+      SENT: null,
+    };
+  }
+
+  return {
+    DRAFT: new Date().toISOString(),
+    IN_PROGRESS: new Date().toISOString(),
+    FINALISED: new Date().toISOString(),
+    SENT: new Date().toISOString(),
+  };
+};
+
+export const Footer: FC<OutboundDetailFooterProps> = ({ draft, save }) => {
+  const navigate = useNavigate();
+  const t = useTranslation('common');
+  const { success } = useNotification();
+
+  return (
+    <AppFooterPortal
+      Content={
+        draft && (
+          <Box
+            gap={2}
+            display="flex"
+            flexDirection="row"
+            alignItems="center"
+            height={64}
+          >
+            <StatusCrumbs
+              statuses={getSupplierRequisitionStatuses()}
+              statusLog={createStatusLog(draft.status)}
+              statusFormatter={getSupplierRequisitionTranslator()}
+            />
+
+            <Box flex={1} display="flex" justifyContent="flex-end" gap={2}>
+              <ButtonWithIcon
```

Comment:
now that this has been replicated a few times.. maybe we should add a button type which has secondary colouring and small font size and `shrinkThreshold` all defaulting to useful values?

Reply:
Ah good call. I supposed it's the same as the `dialog button` style? 🤔